sandbox/registration-processor-mz.properties

```diff
@@ -398,6 +401,12 @@ mosip.registration.processor.lostrid.version=1.0
 
 registration.processor.notification_service_subscriber_secret={cipher}1b1c1a60abf045c34fff60457976178e5c70c949634ad568f5a5510007bfa438
 registration.processor.notification_service_subscriber_callback_url=${mosipbox.public.url}/registrationprocessor/v1/notification/callback/notify
+registration.processor.notification_service_pausedforadditonalinfo_subscriber_secret={cipher}1b1c1a60abf045c34fff60457976178e5c70c949634ad568f5a5510007bfa438
+mosip.regproc.workflow.pausedforadditionalinfo.topic=registration_processor_workflow_paused_for_additional_info_event
+registration.processor.notification_service_pausedforadditonalinfo_subscriber_callback_url=${mosipbox.public.url}/registrationprocessor/v1/notification/callback/notifyPausedForAdditionalInfo
+mosip.regproc.notification_service.biometric_correction.email=RPR_PAUSED_FOR_ADDITIONAL_INFO_EMAIL
+mosip.regproc.notification_service.biometric_correction.sms=RPR_PAUSED_FOR_ADDITIONAL_INFO_SMS
+mosip.regproc.notification_service.biometric_correction.subject=Requesting the additional details for progressing on the application of UIN
```

Comment:
Please add comment for all the properties and leave 1 empty line above email property

Reply:
updated

Reply:
resolved